style_editor.html

```diff
@@ -5746,9 +5746,9 @@
   constructor(T, EFFECT_COLOR, TRANSITION1, TRANSITION2, EFFECT_ARG) {
     super("Trigger transitions on an effect.", arguments);
     this.add_arg("T", "COLOR", "Base color.");
-    this.add_arg("EFFECT_COLOR", "COLOR", "Effect color.");
-    this.add_arg("TRANSITION1", "TRANSITION", "from T to EFFECT_COLOR");
-    this.add_arg("TRANSITION2", "TRANSITION", "from EFFECT_COLOR T");
+    this.add_arg("Effect Color", "COLOR", "Effect color.");
```

Comment:
This will break things. The first argument is the name of the variable the value will be stored in.

Reply:
Oops. That must have been a careless multi-select replace. 
Fixed.